third_party/xla_client/metrics_analysis.cc

```diff
@@ -0,0 +1,204 @@
+#include "tensorflow/compiler/xla/xla_client/metrics_analysis.h"
+
+#include "absl/types/variant.h"
+#include "tensorflow/compiler/xla/xla_client/computation_client.h"
+#include "tensorflow/compiler/xla/xla_client/metrics.h"
+#include "tensorflow/compiler/xla/xla_client/tf_logging.h"
+#include "tensorflow/compiler/xla/xla_client/types.h"
+
+namespace xla {
+namespace metrics {
+
+namespace {
+
+float DivToFloat(std::ldiv_t val, std::int64_t denominator) {
+  return val.quot + (float) val.rem / denominator;
+}
```

Comment:
I am a bit confuse about what this function trying to do, can you provide a simple example? 😄

Reply:
It's basically converting like a `{quot: 3, rem:1}` where `denominator=3` to `3.33` since `=3+1/3`.